tools/run_tests/sanity/check_clang_tidy.sh

```diff
@@ -17,5 +17,5 @@ set -e
 
 make buildtests \
   -j "$(python -c 'import multiprocessing; print multiprocessing.cpu_count()')"
-find src/core src/cpp test/core test/cpp -print0 -name '*.h' -or -name '*.cc' \
+find src/core src/cpp test/core test/cpp -name '*.h' -or -name '*.cc' -print0 \
```

Comment:
Is this necessary? (mostly for my own education)

Reply:
Yep! If the -print0 comes before the -name filters, then it just prints every single file. I was a little surprised that the ordering made such a difference